### ISSUE TITLE
www.github.com is not recognized

### ISSUE THREAD

**Issue by Coronon** [icapps/flutter-icapps-license#19]
In **params.dart** -> `_getLicenseUrl` the check:

```dart
if (url.startsWith(githubDomain)) {
  // ...
}
return null;
```

is not sufficient. Some packages that are hosted on GitHub are not recognized this way. An example is the dart [crypto package](https://pub.dev/packages/crypto), which uses the subdomain 'www'.

It would be awesome if this could be fixed :)

**Comment by vanlooverenkoen** [comment]
This will be fixed in the full refactor. I am working on this right now

**Comment by vanlooverenkoen** [comment]
This is included in our new package license_generator. It is released with version: 1.0.0
